packages/aws-cdk/README.md

```diff
@@ -746,6 +746,32 @@ NOTICES
 If you don’t want to see a notice anymore, use "cdk acknowledge <id>". For example, "cdk acknowledge 16603".
 ```
 
+### `cdk migrate`
```

Comment:
This section is under the "Notices" section. Can you move it up to the "Commands" section?

Reply:
isn't CDK notices the command?

Reply:
no. there is a section header for `Notices` with `cdk notices` and `cdk acknowledge` both under it.  If you view this file as rendered markdown, it is clear that this command is not in the right section.